src/components/Calculator/Calculator.js

```diff
@@ -33,14 +36,12 @@ export default class Calculator extends React.Component {
       return next;
     }
     return '0';
-  }
-
-  render() {
-    return (
-      <div className="calculator">
-        <Screen value={this.display()} />
-        <CalculatorBtns handleBtnClick={this.handleBtnClick} />
-      </div>
-    );
-  }
-}
+  };
+  return (
+    <div className="calculator">
+      <Screen value={display()} />
+      <CalculatorBtns handleBtnClick={handleBtnClick} />
+    </div>
+  );
+};
+export default Calculator;
```

Comment:
- [ ] when I try to divide by 0 currently I get this error

![e](https://user-images.githubusercontent.com/44745855/129413014-3f8a516f-12cb-4500-9a43-60bdca373962.png)
Normally it has to display `error` to the screen please work on that